### ISSUE TITLE
Add plugin images

### ISSUE THREAD

**Issue by MrTimscampi** [jellyfin/jellyfin-ux#43]
* [x] Kodi Sync Queue 
* [x] CoverArtArchive
* [x] MusicBrainz
* [x] Fanart
* [x] OpenSubtitles
* [x] TMDB
* [x] Trakt
* [x] LDAP
* [x] TVDB
* [x] Next PVR
* [x] TV Maze
* [x] ~~TMDB Boxsets~~ (**Should be merged with TMDB**)
* [x] ~~Bookshelf~~ (**Should be split**)
  * [x] Google Books
  * [x] ComicVine
  * [x] EPUB
  * [x] Calibre 
* [x] Auto Organize
* [x] OMDB
* [x] Pushbullet
* [x] TV Headend
* [x] Kitsu
* [x] AniSearch
* [x] AniList
* [x] AniDB
* [x] Slack
* [x] ~~ServerWMC~~ (**Potentially discontinued**)
* [x] Email Notifications
* [x] Playback Reporting
* [x] Reports
* [x] Missing Episode Provider (**Not published**)

**Comment by Shadowghost** [comment]
Sadly most are non-SVG:
* LDAP: https://i1.wp.com/ldap.com/wp-content/uploads/2018/04/ldapdotcom-white-background-with-text-1024x341.png
* AniSearch: https://cdn.anisearch.de/images/anisearch-logo-normal-size.png
* AniList:https://anilist.co/img/icons/android-chrome-512x512.png
* AniDB: https://cdn-us.anidb.net/css/icons/touch/android-chrome-512x512.png
* TV Headend: https://www.minipctv.de/wp-content/uploads/TV-Headend-Logo-ohne-Schriftzug.jpg
* TV Maze: https://static.tvmaze.com/images/tvm-header-logo.png

Ideas for the generic ones:
* Auto-Organize: Dependency-Tree icon
* Email Notification: Just a nice mail icon
* Playback reporting: Playback icon arounda statistics icon

**Comment by MrTimscampi** [comment]
AniList SVG (Thanks @nielsvanvelzen ): https://anilist.co/img/icons/icon.svg

**Comment by h1dden-da3m0n** [comment]
Hi all :wave:, since I currently have most of the JF plugins checked out for the CI push on them I was interested in helping out to add the graphics to the plugins I was wondering how this should be done while honoring the `CC-BY-SA-4.0 License` this Repository and ultimately the SVGs and PNGs would be under?
This question originates from the comments in oddstr13/jellyfin-plugin-repository-manager#3

**Comment by h1dden-da3m0n** [comment]
Okay, I tried out what it would look like once these banners/images are merged and are used within the plugin repositories.
I decided to go with the SVG version of the images and used it as a Banner replacement in the readme as well as added it as a link to it in the `build.yaml` (as a demo see [my fork of the KodiSyncQueue plugin](https://github.com/h1dden-da3m0n/jellyfin-plugin-kodisyncqueue/tree/chore/plugin-images))

I took the liberty to add the license reference  in the SVG based on how it was handled in the SVGs contained in this repositories `branding` folder.

I would really appreciate some feedback if this is a direction I should head or if there is anything I missed (especially License and Image format wise).

FYI @oddstr13
